source/_components/switch.orvibo.markdown

```diff
@@ -11,20 +11,32 @@ logo: orvibo.png
 ha_category: Switch
 ---
 
-The `orvibo` switch platform allows you to toggle your Orvibo S20 Wifi Smart Switch.
+The `orvibo` switch platform allows you to toggle your Orvibo S20 Wifi Smart Switches.
```

Comment:
If rephrasing maybe you should change to the actual name of the products since they are called Sockets by [Orvibo](http://www.orvibo.com/product_en.html)? Just my opinion but might be easier for anyone googling.

Example
```
+The `orvibo` switch platform allows you to toggle your Orvibo S20 Wifi Smart Sockets.
```